lib/index.ts

```diff
@@ -53,6 +54,16 @@ async function buildDepTree(
           'Node.js environment or use `package-lock.json`');
       }
       break;
+    case LockfileType.yarn2:
+      // parsing yarn.lock is supported for Node.js v10 and higher
+      if (getRuntimeVersion() >= 10) {
+        lockfileParser = new Yarn2LockParser();
+      } else {
+        throw new UnsupportedRuntimeError('Parsing `yarn.lock` is not ' +
+          'supported on Node.js version less than 10. Please upgrade your ' +
+          'Node.js environment or use `package-lock.json`');
```

Comment:
in this situation an older  v1 `yarn.lock` would also work? Doesn't make sense to ask to switch to `npm`

Reply:
It will not, as the lock file in v1 is not standard yaml file... And in any case yarn2 only works with node 10 and above so I don't think we will have a problem here